tools/wpt/utils.ts

```diff
@@ -9,14 +9,16 @@ export const {
   wptreport,
   quiet,
   release,
+  build,
   rebuild,
   ["--"]: rest,
   ["auto-config"]: autoConfig,
   binary,
 } = parse(Deno.args, {
   "--": true,
-  boolean: ["quiet", "release", "no-interactive"],
+  boolean: ["build", "quiet", "release", "no-interactive"],
```

Comment:
Use the new `binary` flag for `wpt.ts` instead. If you pass `--binary`, the build will be disabled.